package/src/native.ts

```diff
@@ -145,7 +145,7 @@ export type VideoType = {
   style?: StyleProp<ViewStyle>;
 };
 
-export let Video: React.ComponentType<VideoType> = fail;
+export let Video: React.ComponentType<VideoType>;
```

Comment:
```suggestion
export let Video: React.ComponentType<VideoType> | undefined;
```

Small change, seems to be not that necessary though due to the presence of `isVideoPackageAvailable`. But in future , this allows to just to use the Video component and checking without using the function in my opinion